addons/l10n_in_edi/views/res_config_settings_views.xml

```diff
@@ -1,42 +1,34 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <record id="res_config_settings_view_form_l10n_in_inherit_l10n_in_edi" model="ir.ui.view">
```

Comment:
Could you please write the upgrade script for that?

Reply:
Hello @william-andre
I created an upgrade script for this please have a look at it
PR - https://github.com/odoo/upgrade/pull/4321
Thanks